src/docs/manual/index.html

```diff
@@ -794,21 +794,18 @@ <h2 id="command-line-options">Command-line options</h2>
  <p>See an <a href= "https://randoop.github.io/randoop/manual/method_list_example.txt">
  example</a>.
             <li id="option:omitmethods"><b>--omitmethods=</b><i>regex</i> <code>[+]</code>.
-             Methods whose <code>java.lang.reflect.Method.toString() toString()</code> matches this regex are not
- directly called by test methods. This does not prevent indirect calls to such methods from
- other, allowed methods.
-
- <p>Randoop only calls methods that are specified by one of the <code>--testclass</code>, <code>--classlist</code>, or <code>--methodlist</code> command-line options; the purpose of <code>omitmethods</code>
- --is to override one of those other command-line options.
-
- <p>The regex may match a substring of the method name, unless the regex is anchored with <code>^</code> or <code>$</code>.
-
- <p>Note:
-
- <ul>
-   <li>If a method is inherited without an override, the pattern must match the superclass
-       method.
- </ul>
+             A regex that indicates methods that should not be called directly in generated tests. This does
+ not prevent indirect calls to such methods from other, allowed methods.
+
+ <p>Randoop will not directly call a method whose fully-qualified signature matches the regular
+ expression, or a method inherited from a superclass or interface for which the method matches
+ the regular expression.
+
+ <p>The signature has the form <code>package-name.classname.method-name(argument-list)</code> where
```

Comment:
I'll agree the `--methodlist` format is odd, but there is a difference between giving a loadable name and a regex.  There is a parser for the names that can be more tolerant of spacing differences, while to allow variation in regex input seems like trouble.

Incidentally, the replacement file format in the call-avoidance branch is more like what --methodlist should allow.